spec/forms/csv_upload_form_spec.rb

```diff
@@ -72,8 +72,9 @@
     context 'given a file with 9 keywords and 1 blank' do
       it 'saves 9 keywords with the correct user' do
         user = Fabricate(:user)
+        source = Fabricate(:source)
```

Comment:
Once the constraint is added, there is a high chance that running twice `Fabricate(:source)` or just running once `Fabricate.times(2, :source)` will raise the UNIQUE violation error 🙈 

For that, I recommend to update the Fabricator using the [FFaker](https://github.com/ffaker/ffaker) and its `.unique` util so that it would always return a unique name.

```
name { FFaker::Company.unique.name }
```